plaidml/bridge/openvino/ops/extract_image_patches.cpp

```diff
@@ -74,12 +74,28 @@ void registerExtractImagePatches() {
 
     edsl::Tensor kernel_tensor;
     switch (input_tensor.dtype()) {
+      case DType::FLOAT16:
+        // TODO: Investigate whether there's a way to create fp16 directly
+        kernel_tensor = edsl::cast(create_kernel_tensor<float>(filter_row, filter_col, input_tensor), DType::FLOAT16);
```

Comment:
Should be ok to use the `half` data type here

Reply:
Including half here led to build errors until I set `HALF_ENABLE_CPP11_TYPE_TRAITS`. I'm not sure if `add_compile_definitions` is the right style for such definitions, so let me know if I should be doing this in some other way.